src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -47,6 +47,12 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
   private _vertical = true;
   private _horizontal: 'ltr' | 'rtl' | null;
 
+  /**
+   * Predicate function that can be used to check whether an item should be skipped
+   * by the key manager. By default, disabled items are skipped.
+   */
+  private _skipPredicateFn = (item: T) => item.disabled;
```

Comment:
Call this one `skipPredicate`?

Reply:
This would conflict with the function name that sets the predicate. We could name it `_skipPredicate`, but having the `Fn` suffix feels appropriate IMO.